modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -483,17 +482,25 @@ void ONNXImporter::populateNet(Net dstNet)
          }
         else if (layer_type == "Split")
         {
-            DictValue splits = layerParams.get("split");
-            const int numSplits = splits.size();
-            CV_Assert(numSplits > 1);
+            if(layerParams.has("split"))
+            {
+                DictValue splits = layerParams.get("split");
+                const int numSplits = splits.size();
+                CV_Assert(numSplits > 1);
 
-            std::vector<int> slicePoints(numSplits - 1, splits.get<int>(0));
-            for (int i = 1; i < splits.size() - 1; ++i)
+                std::vector<int> slicePoints(numSplits - 1, splits.get<int>(0));
+                for (int i = 1; i < splits.size() - 1; ++i)
+                {
+                    slicePoints[i] = slicePoints[i - 1] + splits.get<int>(i - 1);
+                }
+                layerParams.set("slice_point", DictValue::arrayInt(&slicePoints[0], slicePoints.size()));
+                layerParams.type = "Slice";
+            }
+            else
             {
-                slicePoints[i] = slicePoints[i - 1] + splits.get<int>(i - 1);
+                layerParams.set("num_split", node_proto.output_size());
+                layerParams.type = "Slice";
```

Comment:
`layerParams.type = "Slice";` is the same for both cases. So let's just move it outside if-else block

Reply:
Sure. I will make this change.